src/command_line_parser.cc

```diff
@@ -2149,10 +2152,6 @@ CLParser::VerifyOptions()
         "Must specify --collect-metrics when using the --metrics-interval "
         "option.");
   }
-  if (params_->streaming && (params_->protocol != cb::ProtocolType::GRPC &&
-                             params_->kind != cb::BackendKind::TRITON_C_API)) {
-    Usage("Streaming is only allowed with gRPC protocol and Triton C API.");
-  }
```

Comment:
Does this if still need to be moved above?

Reply:
I think you might be looking at the old changes.

Reply:
Ah, it was originally up above, and when I filtered the "changes" to only show your recent commits, it was you undoing the change you did earlier where you moved it from up above to down below.